game-core/src/main/java/games/strategy/util/SystemClipboard.java

```diff
@@ -0,0 +1,15 @@
+package games.strategy.util;
```

Comment:
`g.s.ui` (or possibly a new subpackage, `g.s.ui.util`) might be a better location for this type since it is UI-specific.  `g.s.util` should be independent of UI stuff (although it's not today, we probably shouldn't be making it worse).

Reply:
Moved to `g.s.ui`: d743c060a